go/pkg/cas/upload.go

```diff
@@ -42,11 +42,14 @@ var zstdEncoders = sync.Pool{
 	},
 }
 
-// PathSpec specifies a subset of the file system.
-type PathSpec struct {
+// UploadInput specifies a file or directory to upload.
+type UploadInput struct {
 	// Path to the file or a directory to upload.
 	// Must be absolute.
-	Path string
+	Path      string
+	cleanPath string
```

Comment:
Maybe group exported and unexported fields, for people reading the code who just want to figure out what options are available? no strong feelings.

Reply:
done